src/index.html

```diff
@@ -1,12 +1,351 @@
 <!doctype html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+          crossorigin=" ">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page__body">
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item"><a class="nav__list-link is-active" href="#">Apple</a></li>
```

Comment:
Fix code style everywhere
```suggestion
          <li class="nav__list-item">
            <a class="nav__list-link is-active" href="#">Apple</a>
          </li>
          
```